### PR TITLE
Added 8Bit music to the main game

### PR DESCRIPTION
**Pull request by mike4tin**
#114 @anand-bala let me know what you think

### CODE REVIEW

File: src/app/ui/ui.js [hunk 1]
```diff
@@ -3,12 +3,21 @@
  * @module ui/UI
  */
 /**
+ <<<<<<< 8bit_music
```
Comment (anand-bala) [inline src/app/ui/ui.js]: This is the beginning of a merge conflict. This has not been resolved.

File: src/app/ui/ui.js [hunk 2]
```diff
@@ -3,12 +3,21 @@
...
+ * A basic text button.
+ * @param {*} x
+ * @param {number} y
+ * @param {string} text
+ * @param {*} key object associated with button
+ * @param {*} func
+ =======
```
Comment (anand-bala) [inline src/app/ui/ui.js]: This is the divider between incoming (your) changes and current file.

File: src/app/ui/ui.js [hunk 3]
```diff
@@ -3,12 +3,21 @@
...
  * A basic text button. 
  * @param {number} x
  * @param {number} y 
  * @param {string} text 
  * @param {string} key - object associated with button
  * @param {function} func - on click callback function
+ >>>>>>> develop
```
Comment (anand-bala) [inline src/app/ui/ui.js]: This is the end of the previously marked conflict. This one is simple, just remove the first section (from the `8bit_music` line to the `===`), and remove the marking lines

**Review by anand-bala — CHANGES_REQUESTED** [review]
You seem to be committing a new `package-lock.json`. It would be preferable if you replaced this with what was originally in the repo

**Review by anand-bala — APPROVED** [review]

**Comment by anand-bala** [comment]
@mike4tin  This works great, but I am going to hold off merging it for a bit. Thanks a tonne.

**Comment by anand-bala** [comment]
@mike4tin can you fix the merge conflicts?

**Comment by mike4tin** [comment]
Not sure if I can. I can't open it in terminal like I would normally be able to if it was my repo

**Review by anand-bala — APPROVED** [review]

**Pull request closed without merging**